packages/gatsby/src/redux/actions/public.js

```diff
@@ -836,36 +836,39 @@ actions.createNode = (...args) => dispatch => {
  * nodes from a remote system that can return only nodes that have
  * updated. The source plugin then touches all the nodes that haven't
  * updated but still exist so Gatsby knows to keep them.
- * @param {Object} $0
- * @param {string} $0.nodeId The id of a node
+ * @param {object} node A node object. See the "createNode" action for more information about the node object details.
```

Comment:
```suggestion
 * @param {Object} node A node object. See the "createNode" action for more information about the node object details.
```

I'm not sure it matters, but I think we capitalize this elsewhere and it looks like [JSDoc](https://jsdoc.app/tags-param.html#names-types-and-descriptions) capitalizes `Object` in their docs? 🤷‍♂️